pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -652,7 +662,7 @@ func (e *TokensController) getServiceAccount(ns string, name string, uid types.U
 	return nil, nil
 }
 
-func (e *TokensController) getSecret(ns string, name string, uid types.UID, fetchOnCacheMiss bool) (*v1.Secret, error) {
+func (e *TokensController) getSecret(clusterName string, ns string, name string, uid types.UID, fetchOnCacheMiss bool) (*v1.Secret, error) {
 	// Look up in cache
 	obj, exists, err := e.updatedSecrets.GetByKey(makeCacheKey(ns, name))
```

Comment:
The key not including the cluster name suggests the indexer is not safe for use with kcp. How to fix?